candi/tools/functions.sh

```diff
@@ -18,21 +18,28 @@ set -Eeo pipefail
 
 function check_jq() {
     if ! jq --version &>/dev/null; then
-      >&2 echo "ERROR: jq is not installed"
+      >&2 echo "ERROR: jq is not installed. Please install it from https://stedolan.github.io/jq/download"
+      return 1
+    fi
+}
+
+function check_gh() {
+    if ! gh --version &>/dev/null; then
+      >&2 echo "ERROR: gh is not installed. Please install it from https://cli.github.com"
       return 1
     fi
 }
 
 function check_crane() {
     if ! crane version &>/dev/null; then
-      >&2 echo "ERROR: crane is not installed"
+      >&2 echo "ERROR: crane is not installed. Please install it from https://github.com/google/go-containerregistry/tree/main/cmd/crane"
```

Comment:
I'd like to see targets to install crane and yq binaries in the make file. With this, we can easily do
```
make deps
make update-k8s-patch-versions
```

Reply:
yes, good point

Reply:
There is a conflict with the makefile from the main branch because there is the deps target already.
Could you take a look and rearrange your code in line with it (e,g use targets to download dependencies)?

Reply:
resolved